src/main/java/com/aws/greengrass/lambdatransformer/common/models/LambdaEventSource.java

```diff
@@ -20,9 +19,7 @@
     private LambdaEventSourceType type;
 
     private enum LambdaEventSourceType {
-        @JsonProperty("pubsub")
-        PUBSUB,
-        @JsonProperty("iotcore")
```

Comment:
why did you remove the property names? "pubsub" and "iotcore" is correct isn't it?

Reply:
no, they are PUB_SUB and IOT_CORE, respectively